auth/common.cc

```diff
@@ -34,9 +34,9 @@ namespace auth {
 
 namespace meta {
 
-constexpr std::string_view AUTH_KS("system_auth");
-constexpr std::string_view USERS_CF("users");
-constexpr std::string_view AUTH_PACKAGE_NAME("org.apache.cassandra.auth.");
+constinit const std::string_view AUTH_KS("system_auth");
```

Comment:
Nice to use yet another new C++20 feature, but in this case I wonder - how does it make a difference? If the compiler can use constant initialization, wouldn't it do this even without being forced to with "constinit"?

Reply:
It may (and usually will), but this forces an error if it can't.